java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -271,6 +272,63 @@ void executeInstruction(BytecodeCFGBuilder.Instruction instruction) {
         Preconditions.checkState(pop.values.size() == 2, "SWAP needs 2 values on stack");
         programState = pop.state.stackValue(pop.values.get(0)).stackValue(pop.values.get(1));
         break;
+      case IADD:
+      case LADD:
+      case FADD:
+      case DADD:
+      case ISUB:
+      case LSUB:
+      case FSUB:
+      case DSUB:
+      case IMUL:
+      case LMUL:
+      case FMUL:
+      case DMUL:
+      case IDIV:
+      case LDIV:
+      case FDIV:
+      case DDIV:
+      case IREM:
+      case LREM:
+      case FREM:
+      case DREM:
+        handleArithmetic(instruction);
+        break;
+      case INEG:
+      case LNEG:
+      case FNEG:
+      case DNEG:
+        sv = constraintManager.createSymbolicValue(instruction);
+        pop = programState.unstackValue(1);
+        Preconditions.checkState(pop.values.size() == 1, "NEG needs value on stack");
+        programState = pop.state.stackValue(sv).addConstraint(sv, ObjectConstraint.NOT_NULL);
+        break;
+      case ISHL:
+      case LSHL:
+      case ISHR:
+      case LSHR:
+      case IUSHR:
+      case LUSHR:
+        handleArithmetic(instruction);
```

Comment:
As the implementation is the same for those, the cases should be next to the others.